src/bust.py

```diff
@@ -316,18 +316,14 @@ async def send_stats(self, interaction: Interaction) -> None:
 
         # Compute map of submitter --> total length of all submissions
         submitter_to_len = defaultdict(lambda: 0.0)
-        # Default entry in case no valid songs w/ length
-        bot_member = interaction.guild.get_member(self.client.user.id)
-        submitter_to_len[bot_member] = 0.0
 
         errors = False
         for submit_message, attachment, local_filepath in self.bust_content:
             song_len = song_utils.get_song_length(local_filepath)
-            if song_len:
-                submitter = submit_message.author
-                submitter_to_len[submitter] += song_len
-            else:
+            if song_len is None:
                 errors = True
+                song_len = 0
```

Comment:
pedantic, but just to keep the type of each value in this dict the same:

```suggestion
                song_len = 0.0
```